.changeset/serious-roses-stare.md

```diff
@@ -0,0 +1,6 @@
+---
+'example-app': patch
```

Comment:
Don't need to bump this package as it's not a published package.

Reply:
Should be removed now.